### PR TITLE
test code against both rails 4 and 5 (Fixes #32)

### LINKED ISSUE #32: Add automatic tests with Rails 5 to Travis
Travis should test with all available maintained Rails branches and Ruby versions/branches in order to avoid breakage like #31 in the future.

### PR DESCRIPTION
**Pull request by melekes**

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -0,0 +1,7 @@
+appraise "rails-4" do
+  gem "rails", "4.2.7"
```
Comment (oliverklee) [inline Appraisals]: This should be 4.2.7.1 as the .1 is an important security update which most people will have installed.

File: Appraisals [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+end
+
+appraise "rails-5" do
+  gem "rails", "5.0.0"
```
Comment (oliverklee) [inline Appraisals]: This should be 5.0.0.1 as the .1 is an important security update which most people will have installed.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: Appraisals (hunk 1 above)
Comment (akalyaev) [inline Appraisals]: But do we need to be specific for the tests? We do not test rails security here. Are you suggesting changing this file every time there is a new security update?

Comment (oliverklee) [inline Appraisals]: If we want to make sure that the gem works fine with the latest Rails security patches, we should use those versions. I propose using something like ~> 4.2.7, though.

**Pull request merged**